src/component/registry/bpmn-elements-registry.ts

```diff
@@ -54,7 +54,9 @@ export class BpmnElementsRegistry {
     private htmlElementRegistry: HtmlElementRegistry,
     private cssRegistry: CssRegistry,
     private mxGraphCellUpdater: MxGraphCellUpdater,
-  ) {}
+  ) {
+    this.bpmnModelRegistry.onSearchableModelLoad(this.cssRegistry.clear.bind(this.cssRegistry));
```

Comment:
I find this mechanism very difficult to understand what it happens.
I would use the JS `event` in place.

Reply:
I mentioned it in the #1637 issue with a comment explaining why I think this is not the right time to use events for a fix.